_data/trans/en.json

```diff
@@ -14,8 +14,8 @@
     "extensionp5": "Chrome Web Store",
     "extensionp6": "",
     "footercredits": "Made by",
-    "fork": "Fork on GitHub",
-    "forkproject": "Fork the project on GitHub",
+    "contribute": "Contriute on GitHub",
```

Comment:
`contribute` needs to be moved to just after `about`
`pullrequest` to just after `popular`

This applies to all translations